source/installation/index.html

```diff
@@ -6,263 +6,559 @@
-                </div>
-                <div>
-                    <b>Raspberry Pi</b>
-                    <ul>
-                        <li>Home Assistant Operating System</li>
-                        <li>Home Assistant Container</li>
-                        <li>Home Assistant Core</li>
-                    </ul>
-                </div>
+      </div>
+    </div>
+  </div>
+
+  <span class="label easy">Easy</span>
+
+  <h2>DIY with Raspberry Pi</h2>
+  <p>
+    Raspberry Pi, a mini low-cost computer, is one of the most popular platforms
+    for running Home Assistant. If you want to learn how to DIY, this is a good
+    way to start and gain experience.
+  </p>
+
+  <div class="installations-card">
+    <div class="material-card text">
+      <div class="content-container">
+        <div class="image">
+          <img
+            src="https://images.prismic.io/rpf-products/6aebb638-e088-4b28-bbfd-67eae784ca3b_PI_5_FEATURE.jpg?auto=compress%2Cformat&fit=max&w=600&h=400&fm=webp"
+          />
+        </div>
+        <div class="content">
+          <h3>Install Home Assistant on Raspberry Pi</h3>
+          <p>A low-cost DIY solution to get started with Home Assistant</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Assembling a Raspberry Pi setup</li>
+                <li>Flashing a Raspberry Pi</li>
+              </ul>
             </div>
-            {% include assets/chevron_right.html %}
+            <div>
+              <b>TOOLS REQUIRED</b>
+              <ul>
+                <li>Raspberry Pi 3 / 4 / or newer</li>
```

Comment:
```suggestion
                <li>Raspberry Pi 3 or 4 with power supply</li>
```
removing newer, making a note to add it back in as soon as Pi 5 is supported